CHANGELOG.md

```diff
@@ -1,4 +1,7 @@
 # gmusic.js changelog
+
```

Comment:
For consistency with past commits and the other markdown, please remove the trailing newline after the header:

```
# gmusic.js changelog
5.0.0 - ...
```